src/components/ArticleList.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+
+export default function ArticlesList() {
+    const [articles, setArticles] = useState([]);
+  
+    useEffect(() => {
+      getArticles()
+        .then((data) => {
+          setArticles(data);
+        })
+        .catch((err) => {
+          console.error('Failed to fetch Articles:', err);
+        });
+    }, []);
+  
+    return (
+      
+        <ul className="articles-list">
+          {/* {console.log(articles)} */}
```

Comment:
While the request to your api is happening, what is shown to the user? Does the user know something is happening in the background? How might we let them know this?